access-common/src/main/java/edu/unc/lib/dl/ui/controller/LorisContentController.java

```diff
@@ -108,8 +114,10 @@ public void getRegion(@PathVariable("id") String id,
     @RequestMapping("/jp2Proxy/{id}/{datastream}")
     public void getMetadata(@PathVariable("id") String id,
             @PathVariable("datastream") String datastream, HttpServletResponse response) {
+
+        PID pid = PIDs.get(id);
         // Check if the user is allowed to view this object
-        if (this.hasAccess(id, datastream)) {
+        if (this.hasAccess(pid, datastream)) {
 //            try {
```

Comment:
can this stuff be removed?

Reply:
I was generally leaving this class commented out until we work on loris integration again since I'm guessing a lot of parts of it will be the same but haven't been updated yet